.github/workflows/tutorials.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.6, 3.7]
+        python-version: [3.7]
```

Comment:
This implies we'd be removing support for Python 3.6 entirely. @iamtrask, thoughts on that?

If we want to keep 3.6 support for PySyft (but not Crypten), possible we could split the workflows into one for each Python version and exclude the Crypten tests in the 3.6 workflow.

Reply:
Sorry, this should target a separate branch -- the changes are like this because the latest CrypTen master has support for python >= 3.7

Reply:
I'd really ❤️  for it to target the main branch. We want Crypten support, it's just a matter of figuring out how to get it integrated without upending the rest of the apple cart. We're squeezed on one side by Tensorflow (<= 3.7) and on the other side by Crypten. We really could narrow PySyft to 3.7 support only, just want to make sure that isn't going to break something else.

Reply:
Yes - I agree that we should be targeting the main branch. Having multiple separate version of PySyft is really a last resort nuclear option.

Reply:
We have currently two blocking issues to merge this into master, the first is this python 3.6 support, the second is that crypten is now using torch nightly, and it will actually breaks badly if ran with torch==1.4.0

Reply:
What do you think should be the resolution here?
I am thinking that will be a good idea to test with an older version of Crypten (that had support for pytorch 1.4) and target the Syft master? The downside of this is that we are not up-to-date with the Crypten changes.

Reply:
Changed the target to ```master```

Reply:
FWIW we've retargeted to a crypten branch after lots of discussion becuse we need torch nightly
We'll make sure this Openmined:crypten branch is updated regularly